src/util.ts

```diff
@@ -49,6 +49,12 @@ export function errToString(error: any): string {
 	return error.toString();
 }
 
+const outputChannel = vscode.window.createOutputChannel("Azure CosmosDB");
```

Comment:
outputChannnel is a 'disposable' and should be added to context.subscriptions in extension.ts

Reply:
FYI - this comment is not high priority. App service does it this way, too, even though it's not technically correct